tasks/src/main/java/ru/sberbank/school/task02/util/FxResponse.java

```diff
@@ -0,0 +1,33 @@
+package ru.sberbank.school.task02.util;
+
+import java.io.Serializable;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@Getter
+@Builder
+@EqualsAndHashCode
+@AllArgsConstructor
+public class FxResponse implements Serializable {
+    /**
+     * Строковое представление инструмента, например, USD/RUB
+     */
+    private final String symbol;
+    /**
```

Comment:
First sentence of Javadoc is missing an ending period.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1760328760>*